### ISSUE TITLE
add area graph of base quality behind alignment feature glyph

### ISSUE THREAD

**Issue by max-biodatomics** [GMOD/jbrowse#383]
A quality score can be easily visualized without adding extra elements on viewer.It can be implemented in form of bars for a background color. 

Example:
![image](https://f.cloud.github.com/assets/3761270/1581839/05730138-51e0-11e3-86d5-6e46ed4620b1.png)

and 
![image](https://f.cloud.github.com/assets/3761270/1581850/57a04efc-51e0-11e3-949e-c7689f620dd4.png)

**Comment by cmdcolin** [comment]
Could check out https://github.com/elsiklab/qualityglyphs has a per-base quality render as shaded colors

**Comment by keiranmraine** [comment]
I expect this would have a very expensive rendering cost in 30-50x sequence data.  If the read sequence isn't relevant and an per-base aggregate is suitable would it not be better to pre-generate a BigWig file + track of mean or median quality.

**Comment by cmdcolin** [comment]
true. I guess if you really need to see a region with quality scores, you might be able to handle the slowness :)

there is maybe some virtue to seeing values for individual reads though. I thought this recent post from pacbio about igv was really cool for example http://www.pacb.com/blog/igv-3-improves-support-pacbio-long-reads/
